src/screens/Display/index.js

```diff
@@ -122,4 +123,7 @@ const Display = () => {
   );
 };
 
-export default Display;
+export default connect(mapStateToProps, {
+  saveExp,
+  clearRegisters,
```

Comment:
we don't use mapDispatchToProps, it ends up fragmenting code too much and with the new hooks api there is no equivalent.